tests/manage/rgw/test_host_node_failure.py

```diff
@@ -72,6 +84,54 @@ def test_rgw_host_node_failure(
         if noobaa_pod_node is None:
             assert False, "Could not find the NooBaa DB pod"
 
+        # Validate if RGW pod and noobaa-db are hosted on same node
+        # If not, make sure both pods are hosted on same node
+        log.info("Validate if RGW pod and noobaa-db are hosted on same node")
+        rgw_pod_obj = get_rgw_pods()
+        rgw_pod_node_list = [
+            rgw_pod.get().get("spec").get("nodeName") for rgw_pod in rgw_pod_obj
+        ]
+        if not list(set(rgw_pod_node_list).intersection(noobaa_pod_node.name.split())):
+            log.info(
+                "Unschedule other two nodes such that RGW "
+                "pod moves to node where NooBaa DB pod hosted"
+            )
+            worker_node_list = get_worker_nodes()
+            node_names = list(set(worker_node_list) - set(noobaa_pod_node.name.split()))
+            unschedule_nodes(node_names=node_names)
+            ocp_obj = OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+            rgw_pod_obj[0].delete()
+            ocp_obj.wait_for_resource(
+                condition=constants.STATUS_RUNNING,
+                resource_count=len(rgw_pod_obj),
+                selector=constants.RGW_APP_LABEL,
+                timeout=300,
+                sleep=5,
+            )
+            log.info("Schedule those nodes again")
+            schedule_nodes(node_names=node_names)
+
+            # Check the ceph health OK
+            ceph_health_check(tries=90, delay=15)
+
```

Comment:
Also validate that all the ocs pods are in running state

Reply:
done